focus_areas/code_development.md

```diff
@@ -43,11 +43,14 @@ are happening during a certain time period?
 * Question **Issues**: How many issues related to the source code
 are happening during a certain time period?
 
-  * Metric **Issues**(Period): Number of new issues related to the source code.
+  * Metric **Issues**(Period): Number of new issues related to the source code
```

Comment:
I think a better name might be Issues-Opened.

Reply:
We can change it. I just tried to be in sync with other definitions such as "Proposals" or "Commits", and being minimalist with the naming. But I'm happy to change if you feel that's better. Any other opinions?

Reply:
After thinking about it, and because of the reasons mentioned in the previous comments, I agree with finding a more specific name. But to avoid the problems of "open" vs "opened", I'm changing this to "Issues_New", which I hope captures the idea of "issues opened (new) during a certain period". Do you think this is good enough?